src/Algolia.Search.Test/EndToEnd/Analytics/AnalyticsAATest.cs

```diff
@@ -102,7 +98,8 @@ public async Task TestAATesting()
             _index.WaitTask(addAbTest.TaskID);
 
             ABTest abTestToCheck = await BaseTest.AnalyticsClient.GetABTestAsync(abTest.AbTestId.Value);
-            Assert.IsTrue(TestHelper.AreObjectsEqual(abTestToCheck, abTest, "CreatedAt", "Status", "ClickCount", "ConversionCount"));
+            Assert.IsTrue(TestHelper.AreObjectsEqual(abTestToCheck, abTest, "CreatedAt", "Status", "ClickCount",
+                "ConversionCount"));
```

Comment:
Is there a way to prevent / configure line length? Honestly, I've never understood this parameter on linters, most of the time, this leads to worse readability.

Reply:
> Is there a way to prevent / configure line length? Honestly, I've never understood this parameter on linters, most of the time, this leads to worse readability.

Unfortunately not for now with `dotnet format` :(